docs/intro/install.txt

```diff
@@ -57,12 +57,13 @@ You've got three easy options to install Django:
   who have operating systems that distribute Django.
 
 * :ref:`Install an official release <installing-official-release>`. This
-  is the best approach for users who want a stable version number and aren't
-  concerned about running a slightly older version of Django.
+  is the best approach for users who want the latest release.
 
 * :ref:`Install the latest development version
-  <installing-development-version>`. This is best for users who want the
+  <installing-development-version>`. This is good for users who want the
   latest-and-greatest features and aren't afraid of running brand-new code.
+  You might encounter new bugs in the development version. Reporting
+  these helps the development of Django.
```

Comment:
Also, releases of third-party packages are less likely to be compatible with the development version than the latest stable release.